### PR TITLE
Fixes #2401 plots lost during clone

### LINKED ISSUE #2401: 11.1 Residuals vs. Time plot: no refresh
If I change the settings on the observed data tab in a simulation (scaling or weight): Residual vs. time Plots are not refreshed.
I have to close and reopen the simulation to see the changes.

And if I clear the mapping of observed data set: it is not removed from the plot even after reopening the simulation, so I need to create a new plot (this also applies to Predicted vs Observed plots)

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Core/Chart/SimulationTimeProfileChart.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using OSPSuite.Core.Chart;
+
+namespace PKSim.Core.Chart
+{
+   public class SimulationTimeProfileChart : AnalysisChart
```
Comment (msevestre) [inline src/PKSim.Core/Chart/SimulationTimeProfileChart.cs:5]: This was somewhat of a duplcate class. One is abstract in core now. This is the implementation

File: src/PKSim.Infrastructure/Services/ChartTemplatingTask.cs [hunk 2]
```diff
@@ -90,14 +90,14 @@ private void addObservedDataToChart(IChartEditorPresenter chartEditorPresenter,
...
-      public SimulationTimeProfileChart CloneChart(SimulationTimeProfileChart originalChart, IndividualSimulation simulation)
+      public T CloneChart<T>(T originalChart, IndividualSimulation simulation) where T: AnalysisChart
       {
          var clonedChart = _chartFactory.Create(originalChart.GetType()).WithName(originalChart.Name);
          clonedChart.UpdatePropertiesFrom(originalChart, _cloneManager);
 
          initializeFromTemplate(originalChart, clonedChart, simulation);
 
-         return clonedChart.DowncastTo<SimulationTimeProfileChart>();
+         return clonedChart.DowncastTo<T>();
```
Comment (msevestre) [inline src/PKSim.Infrastructure/Services/ChartTemplatingTask.cs:100]: needs to be generic here or at least not so specific

**Review by rwmcintosh — APPROVED** [review]

**Comment by rwmcintosh** [comment]
@msevestre I'm looking at Open-Systems-Pharmacology/OSPSuite.Core#1780 and I can see that the charts are not cloned at all in PI. Is that the correct behavior in the case of PI?

**Comment by msevestre** [comment]
Hard to tell. I would have expected the charts to be cloned as well

**Pull request merged**